includes/validation/class-amp-invalid-url-post-type.php

```diff
@@ -191,22 +191,20 @@ public static function enqueue_post_list_screen_scripts() {
 		wp_enqueue_style(
 			'amp-validation-error-taxonomy',
 			amp_get_asset_url( 'css/amp-validation-error-taxonomy.css' ),
-			array( 'common' ),
+			array( 'common', 'amp-validation-tooltips' ),
 			AMP__VERSION
 		);
 		wp_enqueue_script(
 			'amp-validation-error-detail-toggle',
 			amp_get_asset_url( 'js/amp-validation-error-detail-toggle-compiled.js' ),
-			array(),
+			array( 'wp-dom-ready', 'amp-validation-tooltips' ),
```

Comment:
It looks like `wp-dom-ready` might only be available in Gutenberg (or at least it's merge).

Reply:
It should not be made a script dependency. It is a module dependency.